src/main/java/uk/gov/hmcts/darts/task/runner/impl/ProcessArmResponseFilesAutomatedTask.java

```diff
@@ -28,11 +28,11 @@ public String getTaskName() {
 
     @Override
     protected void runTask() {
-        armResponseFilesProcessor.processResponseFiles();
+        armBatchProcessResponseFiles.batchProcessResponseFiles();
```

Comment:
We don't have to change the name to batch here. This method should be generic one "processResponseFiles" which will be implemented by Single & Batch Processors. We can use the same approach as Push mechanism.